src/css/App.css

```diff
@@ -0,0 +1,69 @@
+/** App styles */
+main {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(3, 1fr);
```

Comment:
These areas should be described explicitly instead of relying on numbers (i.e., "row 2 col 1" to describe the area the west field will be loaded into)